recipes/hcrystalball/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hcrystalball" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d46d60b158af1a4f820444bde8028f872f69494a70af95e25fc6fc439ec9b937
+
+build:
```

Comment:
Add `skip: True  # [win]` for now, since Windows isn't tested yet?

Reply:
It's `noarch: python` so it's only built on Linux and it's not possible to declare a platform dependence on noarch packages.